bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
I believe you're right. I think we're safe here, and we can follow up with removing this and any other unused keys from the data dict in a separate PR.